.github/actions/setup-node/action.yml

```diff
@@ -9,6 +9,10 @@ inputs:
     description: "Extra flags to pass to the pnpm install."
     required: false
     default: ""
+  no-install:
+    description: "Don't run the install step."
+    required: false
+    default: false
```

Comment:
Inverting this would be much nicer to my brain 😄 . (i.e. make the input `package-install` and set the default to `true`)